dhall-openapi/hie.yaml

```diff
@@ -0,0 +1,7 @@
+cradle:
```

Comment:
I think this is the first `hie.yaml` in this project.

What's the purpose of this file? Which tools can use it?

Reply:
hie.yaml is a file that is used by https://github.com/mpickering/hie-bios (required for `ghcide` or the new integration point: https://github.com/haskell/haskell-language-server)
I can remove it but it won't harm.

Sorry for the `sandbox` folder. I should have checked PR further. Will remove that folder.

Reply:
I have squashed both commits, remove both sandbox and `hie.yaml`